cypress/support/types/backoffice/product-management.ts

```diff
@@ -0,0 +1,24 @@
+import { User, ProductClass, Store } from './shared';
+
+export interface ProductManagementStaticFixtures {
+  defaultPassword: string;
+}
+
+export interface ProductManagementDynamicFixtures {
+  rootUser: User;
+  productClass: ProductClass;
+  product: Product;
+  storeAT: Store;
+  storeDE: Store;
+}
+
+export interface Product {
```

Comment:
please move interface to shared folder
Use the current file only for fixture definitions.